php-practice_2.php

```diff
@@ -0,0 +1,93 @@
+<?php
+// Q1 tic-tac問題
+for ($i=1; $i < 101; $i++) { 
```

Comment:
[nits]
`=`の前後は空白を開けるようにしましょう！
```suggestion
for ($i = 1; $i < 101; $i++) { 
```

Reply:
ご指摘頂いた通り、=の前後に空白を設けました。